_services_projects/alaska-dhss.md

```diff
@@ -3,6 +3,7 @@ agency: Alaska Department of Health and Social Services
 title: Modernizing Access to Healthcare
 subtitle: Modernizing public benefit eligibility systems
 permalink: /what-we-deliver/alaska-dhss/
+redirect_from: /project/alaska-dhss/
```

Comment:
I don't think this is the right redirect (got this from the gDoc). If I do a redirect from https://modularcontracting.18f.gov/projects/ak-medicaid/, will it work, or does it only work if it's redirecting from within this domain?

Reply:
Yeah, this redirect won't work yet, but we'll leave it in for when we work on all the redirects for the modular contracting site.